### ISSUE TITLE
content-type parameter not working

### ISSUE THREAD

**Issue by IGx89** [Azure/azure-storage-azcopy#475]
### Which version of the AzCopy was used? 
10.1.2

### Which platform are you using? (ex: Windows, Mac, Linux)
Windows and Linux (confirmed issue on both)

### What command did you run?
copy ThirdPartyNotice.txt https://mystorageaccount.blob.core.windows.net/$web/SomeFile.application --content-type=application/x-ms-application 

Generated request:
```
   PUT https://mystorageaccount.blob.core.windows.net/$web/SomeFile.application 
   Content-Length: [15910]
   User-Agent: [AzCopy/10.1.2 Azure-Storage/0.6 (go1.10.8; Windows_NT)]
   X-Ms-Blob-Cache-Control: []
   X-Ms-Blob-Content-Disposition: []
   X-Ms-Blob-Content-Encoding: []
   X-Ms-Blob-Content-Language: []
   X-Ms-Blob-Content-Md5: []
   X-Ms-Blob-Content-Type: [text/plain; charset=utf-8]
   X-Ms-Blob-Type: [BlockBlob]
   X-Ms-Client-Request-Id: [2346d9d8-0e00-4f93-58d0-bf65e87bbeed]
   X-Ms-Version: [2018-03-28]
```

### What problem was encountered?
Content-Type stored was text/plain (the default), not application/x-ms-application. The content-type parameter doesn't seem to be taking effect

### How can we reproduce the problem in the simplest way?
Create an Azure blob storage account and attempt to copy a file to it while using the content-type parameter

### Have you found a mitigation/solution?
Manually changing the content-type using Storage Explorer afterwards

**Comment by zezha-msft** [comment]
Hi @IGx89, thanks for reaching out! And sorry for the inconvenience.

It's a known issue that's we'll fix shortly. In the mean time, please add the flag "--no-guess-mime-type" to your command. Thanks!

**Comment by IGx89** [comment]
Ahhh, thanks! That workaround works in the meantime, appreciate it.

**Comment by zezha-msft** [comment]
@IGx89 thanks for your understanding. 😄

**Comment by zezha-msft** [comment]
This has been fixed in [10.3.0](https://github.com/Azure/azure-storage-azcopy/releases/tag/v10.3.0)!
